backend/src/main/java/com/bakdata/conquery/resources/admin/ui/model/UIContext.java

```diff
@@ -1,35 +0,0 @@
-package com.bakdata.conquery.resources.admin.ui.model;
-
-import java.util.Collection;
-
-import com.bakdata.conquery.models.worker.DatasetRegistry;
-import com.bakdata.conquery.models.worker.WorkerInformation;
-import com.bakdata.conquery.resources.ResourceConstants;
-import freemarker.template.TemplateModel;
-import lombok.Getter;
-import lombok.RequiredArgsConstructor;
-
-@RequiredArgsConstructor
-public class UIContext {
```

Comment:
Bist du dir sicher, dass der UIContext nicht verwendet wird? In manchen Templates wurde workers glaube ich verwndet.